### ISSUE TITLE
No printable version of the non-thesis version

### ISSUE THREAD

**Issue by ggerard** [berb/diploma-thesis#9]
Doesn't seem to be a link to download a PDF or HTML suitable version for printing.

**Comment by berb** [comment]
Yes, I know. Although there are only minor spelling differences between both versions for the moment, I'm going to work on a way to export the HTML-based community version to EPUB and PDF.

**Comment by pvorb** [comment]
Why don't you convert the HTML documents by using [Pandoc](http://johnmacfarlane.net/pandoc/)?

Simply transfer the html pages to markdown with `pandoc -o 011_motivation.html 011_motivation.html`. Do that for each HTML file, then concatenate the files into one large text file (`thesis.txt`) and generate the PDF and the EPUB files: `pandoc -o thesis.pdf thesis.txt` and `pandoc -o thesis.epub thesis.txt`.

**Comment by berb** [comment]
Although this works to some extent, this approach removes structured content such as images, tables, footnotes, etc. 
I am not too happy with that.

**Comment by pvorb** [comment]
You're right. I didn't take an in-depth look at the more “advanced” markup. How did you write the original version of the thesis? The HTML looks a bit like an export from LaTeX, but I'm not quite sure. I might be better to use those TeX (or whatever) sources to generate the HTML, PDF and EPUB sources like you did, when you wrote the original thesis.
